disnake/embeds.py

```diff
@@ -302,6 +313,13 @@ def __bool__(self) -> bool:
             )
         )
 
+    def __eq__(self, other: Any) -> bool:
+        return isinstance(other, Embed) and all(
```

Comment:
This should use `color` instead of `_color` to take default colors into account, otherwise sending an embed and comparing it to the one in the created message would return false.
If color is handled separately, I believe the `if ... is not MISSING` part here won't be necessary anymore, since color is the only attribute that may be set to `MISSING`.